src/components/Logout.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom'
+
+class Logout extends Component {
+
+  logout() {
```

Comment:
this doesn't quite look complete to me, but a logout button is definitely a good idea.  It should be a connected component and we'll also need to update `authenticated` to false